generic/tests/netperf.py

```diff
@@ -94,6 +94,9 @@ def env_setup(session, ip, user, port, password):
                 ignore_status=True)
         ssh_cmd(session, "echo 2 > /proc/sys/net/ipv4/conf/all/arp_ignore")
         ssh_cmd(session, "echo 0 > /sys/kernel/mm/ksm/run", ignore_status=True)
+        ssh_cmd(session, "echo 0 > /proc/sys/kernel/watchdog", ignore_status=True)
+        ssh_cmd(session, "echo 0 > /proc/sys/kernel/nmi_watchdog", ignore_status=True)
+        ssh_cmd(session, "setenforce 1", ignore_status=True)
```

Comment:
Hi wenli, a question for you, why do you set ignore_status=True here? Should we raise an exception if any error?

Reply:
I update these all command to param env_setup_cmd. please help a review.

Reply:
> Hi wenli, a question for you, why do you set ignore_status=True here? Should we raise an exception if any error?

Any response about it?

Reply:
Since env_setup for host/client/external host, some commands may not support or different, use ignore_status=True to ignore such error.